joschahenningsen/TUM-Live
# 1214 playback at t 0

### Motivation and Context
fixed #1214 

### Description
Distinguish the behavior of the watchProgress function when the parameter t is set to 0 and when it is not set, so that it matches the description [in the comment](https://github.com/TUM-Dev/gocast/blob/109be9a6e6dd6c115ae9fffdc37992a7af1f0236/web/ts/TUMLiveVjs.ts#L314):
> If query parameter 't' is specified, the timestamp given by 't' will be used.

### Steps for Testing


1. Clear website cache
2. Log in
3. Navigate to a recording video
4. Jump to anywhere in the video other than the start
5. Refresh the page, note that the watch progress has been recorded and can be restored, so that the video resumes at the last progress
6. Add parameter "t=0" to current url and visit it
7. See the video plays from the start
